fairseq/data/audio/multi_modality_dataset.py

```diff
@@ -266,7 +266,7 @@ def __getitem__(self, index):
         return sample
 
     def collater(self, samples, pad_to_length=None):
-        return self.dataset.collater(samples, pad_to_length)
+        return self.dataset.collater(samples, pad_to_length=pad_to_length)
```

Comment:
Thanks Marco. Do you have a case example that failed in the old implementation?

Reply:
yes, I added a UT for this as well. Without this fix the UT fails with an exception. It happens when we have a multilingual text dataset.